packages/cactus-plugin-ledger-connector-iroha/src/main/json/openapi.json

```diff
@@ -0,0 +1,380 @@
+              "description": "The amount of milliseconds to wait for a transaction receipt beforegiving up and crashing. Only has any effect if the invocation type is SEND",
+              "minimum": 0,
+              "default": 60000,
+              "nullable": false
+            },
+            "keychainId": {
+              "type": "string",
+              "description": "The keychainId for retrieve the contracts json.",
+              "minLength": 1,
+              "maxLength": 100
+            }
+          }
+        },
+        "InvokeContractV1Response": {
+          "type": "object",
+          "required": ["success"],
+          "properties": {}
+        },
+        "PrometheusExporterMetricsResponse": {
+          "type": "string",
+          "nullable": false
+        }
+      }
+    },
+    "paths": {
+      "/api/v1/plugins/@hyperledger/cactus-plugin-ledger-connector-iroha/run-transaction": {
+        "post": {
+          "x-hyperledger-cactus": {
+            "http": {
+              "verbLowerCase": "post",
+              "path": "/api/v1/plugins/@hyperledger/cactus-plugin-ledger-connector-iroha/run-transaction"
+            }
+          },
+          "operationId": "runTransactionV1",
+          "summary": "Executes a transaction on a Iroha ledger",
+          "parameters": [],
+          "requestBody": {
+            "content": {
+              "application/json": {
+                "schema": {
+                  "$ref": "#/components/schemas/RunTransactionRequest"
+                }
+              }
+            }
+          },
+          "responses": {
+            "200": {
+              "description": "OK",
+              "content": {
+                "application/json": {
+                  "schema": {
+                    "$ref": "#/components/schemas/RunTransactionResponse"
+                  }
+                }
+              }
+            }
+          }
+        }
+      },
+      "/api/v1/plugins/@hyperledger/cactus-plugin-ledger-connector-iroha/invoke-contract": {
+        "post": {
+          "x-hyperledger-cactus": {
+            "http": {
+              "verbLowerCase": "post",
+              "path": "/api/v1/plugins/@hyperledger/cactus-plugin-ledger-connector-iroha/invoke-contract"
+            }
+          },
+          "operationId": "invokeContractV1",
+          "summary": "Invokes a contract on a Iroha ledger",
+          "parameters": [],
+          "requestBody": {
+            "content": {
+              "application/json": {
+                "schema": {
+                  "$ref": "#/components/schemas/InvokeContractV1Request"
+                }
+              }
+            }
+          },
+          "responses": {
+            "501": {
+              "description": "Not implemented",
+              "content": {
+                "text/plain": {
+                  "schema": {
+                    "type": "object",
+                    "properties": {
+                      "message": {
```

Comment:
@hxlaf Please double check the openapi documentation on this but from the top of my head I think defining properties for a `text/plain` type response is not valid/correct because in plain text all you get back from the server is a single piece of string that is not JSON formatted (or if it is then we should change the content type to json accordingly - one or the other)

Reply:
Referred to the openapi documentation and corrected! Thank you for pointing out.